MatrixSDK/JSONModels/MXRoomCreationParameters.m

```diff
@@ -79,4 +79,18 @@ - (NSDictionary*)JSONDictionary
     return dictionary;
 }
 
+
+#pragma mark - Factory
+
++ (instancetype)parametersForDirectRoomWithUser:(NSString*)userId
+{
+    MXRoomCreationParameters *roomCreationParameters = [MXRoomCreationParameters new];
+    roomCreationParameters.inviteArray = @[userId];
+    roomCreationParameters.visibility = kMXRoomDirectoryVisibilityPrivate;
```

Comment:
I agree to set this value here to clarify it (but this is the default value on server side when visibility is undefined)